docs/agents/browser.md

```diff
@@ -0,0 +1,123 @@
+title: Browser SDK
+description: Ship browser metrics to Sematext Experience.
+
+Sematext [Browser SDK](https://github.com/sematext/browser-sdk) is an open-source framework used to ship web application metrics related to user interaction. Using [Browser SDK](https://github.com/sematext/browser-sdk) allows gathering metrics related to:
```

Comment:
>  used to ship web application metrics 

... what about static web sites, for example?  @amir-hadzic how do we phrase this typically?  We've had this in N places before.

Reply:
In most cases we say "website" or more specifically "website and web app". So I think we could change this to "website and web app" and use "website" in the rest of the document.